storage/client_split_test.go

```diff
@@ -412,6 +412,64 @@ func TestStoreZoneUpdateAndRangeSplit(t *testing.T) {
 	}
 }
 
+// TestMaxBytesChangeAndRangeSplit tests a scenario where a new zone
+// config that sets the max bytes is updated. The new zone config also
+// triggers a range split.
+//
+// TODO(kkaneda): Tweak this test if a new range can get a new value
+// of max bytes without another zone config change.
+func TestMaxBytesChangeAndRangeSplit(t *testing.T) {
```

Comment:
I still think this name is not descriptive - changing max bytes does not split the range.